src/vfs.c

```diff
@@ -593,7 +513,7 @@ static uint32_t vfsDatabaseGetPageSize(struct vfsDatabase *d)
 
 	/* The page size is stored in the 16th and 17th bytes of the first
 	 * database page (big-endian) */
-	return vfsParsePageSize(vfsGet16(&page[16]));
+	return vfsParsePageSize(ByteGetBe16(&page[16]));
```

Comment:
I find the new implementations easier to follow (self-contained, fewer lines of code) -- since they're in a header file now I could add some quick unit tests against the old implementation.